scripts/microbe_stage/organelle_table.as

```diff
@@ -464,15 +464,15 @@ void setupOrganelles(){
     chemoplast.mesh = "chemoplast.mesh";
     chemoplast.chanceToCreate = 1;
     chemoplast.prokaryoteChance = 0;
-    chemoplast.mpCost = 40;
+    chemoplast.mpCost = 45;
     chemoplast.initialComposition = {
         {"phosphates", 5},
         {"ammonia", 5}
     };
     chemoplast.components = {
         processorOrganelleFactory(1.0f),
     // Chemoplast takes 2 hexes, so allowed storage of 2 cytooplasm
-    storageOrganelleFactory(10.0f)
+    storageOrganelleFactory(0.0f)
```

Comment:
if it stores 0 just remove that factory from the list,

Reply:
my bad i didn't restore that value